db/migrate/20190912143132_add_release_field_to_os.rb

```diff
@@ -0,0 +1,40 @@
+class AddReleaseFieldToOs < ActiveRecord::Migration[5.2]
+  class Operatingsystem < ApplicationRecord
+    self.inheritance_column = nil
+  end
+
+  def up
+    add_column :operatingsystems, :release, :string, :limit => 16, :default => "", :null => false
+
+    ok = true
+    Operatingsystem.all.each do |os|
```

Comment:
I would rather see it done directly through sql:
```ruby
Operatingsystem.all.pluck(:id, :minor).each do |(id, minor)|
  if minor&.include?('.')
    minor_release = minor.split('.')
    Operatingsystem.where(id: id).update_all(minor: minor_release[0], release: minor_release[1..-1].join('.'))
    # see comment below for the OK check.
  end
end
```
Do you think it would be possible to use something from this to improve the readability a bit?